_posts/2022-03-23-#21778.md

```diff
@@ -10,14 +10,39 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+Historically Bitcoin Core has used a very ["non-standard" toolchain](https://github.com/bitcoin/bitcoin/blob/master/contrib/macdeploy/README.md) for producing its macOS release binaries. This is primarily due to there not being any "official" way to build macOS binaries, using completely open source software, on a Linux system. The only Apple-sanctioned way to build macOS binaries is using the Xcode toolchain, running on macOS hardware.
+
+Using a non-standard toolchain is not ideal for a number of reasons:
+    * We are the only user. No one else is testing things / finding bugs / making improvements.
+    * Creating our toolchain is reliant on Apple (infrequently) [releasing source code](https://opensource.apple.com/source/) for the tools we need.
+    * It's reliant on 3rd-parties patching those sources. i.e https://github.com/tpoechtrager/cctools-port
+    * It's complicated.
+
+The most promising alternative toolchain for producing macOS binaries on Linux, has been LLVMs `clang` + `lld`. However until recently, `lld` was unable to link large / complex programs. Much work has been done over the past few years [to re-write the Mach-O backend](https://reviews.llvm.org/rG03f43b3aca3) for `lld`, to where it is now able to link large, complex programs, i.e Chromium, as well as self-host. The new backend has now also [become the default](https://lists.llvm.org/pipermail/llvm-dev/2021-January/147665.html), [as of LLVM 13](https://reviews.llvm.org/D95204).
+
+[#21778](https://github.com/bitcoin/bitcoin/pull/21778) is a, not quite working yet, attempt to transition our macOS toolchain over to using a very vanilla, `clang` + `lld`, as opposed to our current, custom built toolchain.
+
 ## Questions
+
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
--->
+
+2. Can you think of a reason why we might want to continue using our current macOS toolchain, rather than switch?
+
+3. Did you try performing a macOS cross compile? Did it work? If not, what problems did you run into?
```

Comment:
It may be helpful to link to instructions cross compile for macOS again (otherwise people's answers will almost certainly be "no").